src/sagemaker/jumpstart/curated_hub/accessors/public_model_data.py

```diff
@@ -0,0 +1,111 @@
+        studio_specs: Dict[str, Dict[str, Any]],
+    ):
+        self._region = region
+        self._bucket = get_jumpstart_content_bucket(region)
+        self.model_specs = model_specs
+        self.studio_specs = studio_specs  # Necessary for SDK - Studio metadata drift
+
+    def get_bucket_name(self) -> str:
+        """Retrieves s3 bucket"""
+        return self._bucket
+
+    def get_inference_artifact_s3_reference(self) -> S3ObjectLocation:
+        """Retrieves s3 reference for model inference artifact"""
+        return create_s3_object_reference_from_uri(
+            self._jumpstart_artifact_s3_uri(JumpStartScriptScope.INFERENCE)
+        )
+
+    def get_training_artifact_s3_reference(self) -> S3ObjectLocation:
+        """Retrieves s3 reference for model training artifact"""
+        return create_s3_object_reference_from_uri(
+            self._jumpstart_artifact_s3_uri(JumpStartScriptScope.TRAINING)
+        )
+
+    def get_inference_script_s3_reference(self) -> S3ObjectLocation:
+        """Retrieves s3 reference for model inference script"""
+        return create_s3_object_reference_from_uri(
+            self._jumpstart_script_s3_uri(JumpStartScriptScope.INFERENCE)
+        )
+
+    def get_training_script_s3_reference(self) -> S3ObjectLocation:
+        """Retrieves s3 reference for model training script"""
+        return create_s3_object_reference_from_uri(
+            self._jumpstart_script_s3_uri(JumpStartScriptScope.TRAINING)
+        )
+
+    def get_default_training_dataset_s3_reference(self) -> S3ObjectLocation:
+        """Retrieves s3 reference for s3 directory containing model training datasets"""
+        return S3ObjectLocation(self.get_bucket_name(), self._get_training_dataset_prefix())
+
+    def _get_training_dataset_prefix(self) -> str:
+        """Retrieves training dataset location"""
+        return self.studio_specs["defaultDataKey"]
+
+    def get_demo_notebook_s3_reference(self) -> S3ObjectLocation:
+        """Retrieves s3 reference for model demo jupyter notebook"""
+        framework = get_model_framework(self.model_specs)
+        key = f"{framework}-notebooks/{self.model_specs.model_id}-inference.ipynb"
+        return S3ObjectLocation(self.get_bucket_name(), key)
+
+    def get_markdown_s3_reference(self) -> S3ObjectLocation:
+        """Retrieves s3 reference for model markdown"""
+        framework = get_model_framework(self.model_specs)
+        key = f"{framework}-metadata/{self.model_specs.model_id}.md"
+        return S3ObjectLocation(self.get_bucket_name(), key)
+
+    def _jumpstart_script_s3_uri(self, model_scope: str) -> str:
+        """Retrieves JumpStart script s3 location"""
+        return script_uris.retrieve(
+            region=self._region,
+            model_id=self.model_specs.model_id,
+            model_version=self.model_specs.version,
+            script_scope=model_scope,
+            tolerate_vulnerable_model=True,
+            tolerate_deprecated_model=True,
```

Comment:
why is this?

Reply:
Copy-and-pasted from the POC. Should I remove?